aspnetcore/tutorials/grpc/grpc-start/sample/GrpcGreeterClient/GrpcGreeterClient.csproj

```diff
@@ -7,8 +7,8 @@
 
   <ItemGroup>
     <PackageReference Include="Google.Protobuf" Version="3.8.0" />
-    <PackageReference Include="Grpc.Net.Client" Version="0.1.22-pre2" />
-    <PackageReference Include="Grpc.Tools" Version="1.22.0-pre1" PrivateAssets="All" />
+    <PackageReference Include="Grpc.Net.Client" Version="0.2.23-dev201908080701" />
```

Comment:
I'm assuming we don't plan to publish this till preview9 ships?

Reply:
ye